lib/Conversion/TTIRToTTIRDecomposition/TTIRToTTIRDecomposition.cpp

```diff
@@ -775,6 +776,123 @@ class GetDimensionSizeToConstantConversionPattern
   }
 };
 
+struct ArangeForceLastDimensionPattern
+    : public OpConversionPattern<ttir::ArangeOp> {
```

Comment:
This is a bit complex decomposition, so I would suggest that we start writing tests specifically for decompostions, hence we can create a test folder under:
test/ttmlir/Dialect/TTIR/Decompositions/

We can use this to write decompostion specific compiler tests by running just the decomposition pass and not the whole pipeline. Here we can test all the little details that are introduced with a specific decompositon.

Reply:
I've added a test here